unitTests/MockCookie.ts

```diff
@@ -0,0 +1,52 @@
+function mapToObject<T = any>(list: string[], predicate: (value: string) => [string, string]): T {
```

Comment:
This new file was introduced because CookieUtilsTest replaced `document.cookie` with an incomplete mock, which only allowed a single cookie to be defined and affected other unit tests.